Example/TABTestKit_ExampleUITests/ExampleContext.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  ExampleContext.swift
+//  TABTestKit_ExampleUITests
+//
+//  Created by Suyash Srijan on 05/11/2018.
+//  Copyright © 2018 CocoaPods. All rights reserved.
+//
+
+import Foundation
+
```

Comment:
Unneeded space (pedantic I know sorry, just thinking this is public facing so nice to have everything formatted nicely)